src/app/core/resource-policy/resource-policy.service.ts

```diff
@@ -221,4 +233,43 @@ export class ResourcePolicyService {
     return this.dataService.searchBy(this.searchByResourceMethod, options, useCachedVersionIfAvailable, reRequestOnStale, ...linksToFollow);
   }
 
+  /**
+   * Update the target of the resource policy
+   * @param resourcePolicyId the ID of the resource policy
+   * @param resourcePolicyHref the link to the resource policy
+   * @param uuid the UUID of the target to which the permission is being granted
+   * @param type the type of the target (eperson or group) to which the permission is being granted
+   */
+  updateTarget(resourcePolicyId: string, resourcePolicyHref: string, uuid: string, type: string): Observable<RemoteData<any>> {
```

Comment:
I'm not seeing new tests in `resource-policy.service.spec.ts` that correspond to this new method.  Could we add tests for it?